doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Timers in node.js
+layout: docs.hbs
+---
+
+# Timers in node.js and beyond
+
+Timers are a collection of global functions in node.js that allow you to 
```

Comment:
Keep in mind you can actually do `require('timers')` to get at other stuff. They are just exposed the global object to mirror browsers. As such, I think there could be a better lead-in.